packages/client/hmi-client/src/services/models/simulation-service.ts

```diff
@@ -245,6 +248,25 @@ export async function makeEnsembleCiemssCalibration(params: EnsembleCalibrationC
 	}
 }
 
+export async function createSimulationAssets(
+	projectId: string,
+	simulationId: string,
+	simulationType: SimulationType,
+	newName: string | null,
+	assetId?: string
+) {
+	try {
+		const response: AxiosResponse<TerariumAsset[]> = await API.post(
+			`/simulations/${simulationId}/create-assets-from-simulation/${projectId}?name=${newName}&simulation-type=${simulationType}${assetId ? `&asset-id=${assetId}` : ''}`
```

Comment:
Your POST request is rather unconventional that it is like a GET request, it may be better to have a proper POST-body instead of this URL string logic which can have encoding issues.

Reply:
fair, will make those changes

Reply:
changes made